.travis.yml

```diff
@@ -20,6 +20,9 @@ matrix:
 # These directories are cached to S3 at the end of the build
 cache:
   directories:
+    # The following two directories are for zipkin-web, which gets dependencies via NPM
+    - $HOME/.npm
+    - zipkin-web/node_modules
```

Comment:
Since `~/.npm` is cached, I don't think it'll have a big impact; I expect the majority of the build time to come from downloading the packages from NPM. The only thing I can think of that can slow us down is if any compilation takes place, say if there are native extensions to build.